apps/main/src/dao/components/PageVeCrv/components/FieldDatePicker.tsx

```diff
@@ -2,11 +2,12 @@ import { useMemo, useState } from 'react'
 import { styled } from 'styled-components'
 import type { FormType, VecrvInfo } from '@/dao/components/PageVeCrv/types'
 import { CurveApi } from '@/dao/types/dao.types'
-import { formatDisplayDate, toCalendarDate } from '@/dao/utils/utilsDates'
+import { toCalendarDate } from '@/dao/utils/utilsDates'
 import type { DateValue } from '@internationalized/date'
 import Button from '@ui/Button'
 import DatePicker from '@ui/DatePicker'
 import { Chip } from '@ui/Typography'
+import { formatDate } from '@ui/utils'
```

Comment:
Can you please update the incorrect comment here? As you said, the format printed is different
```
en-US: Dec 19, 2025
en-UK: 19 Dec 2025
pt-BR: 19 de dez. de 2025
nl-NL: 19 dec 2025
hi: 19 दिस॰ 2025
zh: 2025年12月19日
```
https://github.com/curvefi/curve-frontend/blob/b5d95f7f5fbcd7e6c9a2b5b585beab86b90946d5/packages/ui/src/utils/utilsDate.ts#L58